colin/checks/dockerfile/from_tag.py

```diff
@@ -4,11 +4,12 @@
 class FromTagCheck(InstructionCheck):
 
     def __init__(self):
-        super().__init__(name="is_tag_latest",
+        super().__init__(name="is_tag_not_latest",
                          message="",
                          description="",
                          reference_url="https://docs.docker.com/engine/reference/builder/#from",
                          tags=["from", "dockerfile", "latest"],
                          instruction="FROM",
-                         regex=".*/latest$",
+                         value_regex=".*/latest$",
```

Comment:
`FROM fedora` is also bad, we should catch that one as well

Reply:
Yes, there is a TODO for this on the next line:
>  # TODO: Does not check if there is no tag => use ImageName parsing.

It will not be hard to do it.

Reply:
Done.